recipes/rt1/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "rt1" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "f95c9acca34d22e04b5412687b856dfe1332405e0fc749c178f0e9f24ce84179" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win]
```

Comment:
package was never tested under windows so far.
Could not guarantee that it will work. Therefore the `skip: True` was set.